### DISCUSSION TITLE
What stops the sharpies from drying out?

### DISCUSSION THREAD

**Discussion by Dr-Ari-Gami** [Sakati84/3DPrintColorizer#32] [Q&A]
This seems like an awesome idea, but what stops the sharpies from drying out?

**Comment by Sakati84** [comment]
For me they don't dry out for long prints (18 hours) and if they still dry out afterwarts put them in isopropyl alcohol for 30 seconds.

**Comment by doggohusk** [comment]
technically nothing is stopping it from drying it but just simply taking the sharpies out and and puttign the cap on after every normal time print(1 Minute - 10Hours) as sakati says it might even go for longer
